### ISSUE TITLE
Skewed Lognormal Distributions

### ISSUE THREAD

**Issue by kka1996** [SALib/SALib#584]
Hi everyone, 

I am trying to run a sobol analysis on a dataset which is distributed lognormally but with a skewness in it. Consequently, I am not able to represent the data accurately with the ln-spaced mean and standard deviation. 
This is how the data looks like in the original feature-space: 
![grafik](https://github.com/SALib/SALib/assets/141942107/7de0e704-7cc9-4650-9dea-31f68ca89a57)

This is how the data looks like once I apply ln to the x-axis: 
![grafik](https://github.com/SALib/SALib/assets/141942107/509fb695-dcaa-4e1b-af34-84744f07dd97)

One can clearly see that the data has some skewness in it. If I simply take the mean and standard deviation of the log-transformed data and try to sample from the distribution I don't get any good samples. I also tried calculating the mean and standard deviation using the location, scale and shape parameter which I got from from scipy.stats.lognorm.fit(). Am I missing something here, would be great if someone is able to help. 

Best

**Comment by ConnectedSystems** [comment]
Just to check, are you specifying the relevant factors as having a log-normal distribution?

See: https://salib.readthedocs.io/en/latest/user_guide/advanced.html#generating-alternate-distributions

Otherwise, I suggest trying to transform the data to a gaussian distribution.

**Comment by kka1996** [comment]
@ConnectedSystems yes, I went through the referred page and for the given data I calculated the ln-spaced mean and standard deviation. However, for skewed lognormal distributions these two parameters are not sufficient to describe the distribution.

Regarding the transformation: I am not quite sure how this will solve the problem. I tried to fit a normal distribution to the data in the original space as well, however, this (similar to fitting a lognormal distribution) did not give a good fit. Thus, I believe that there needs to be a way to include the skewness of the distribution as well in the input parameters. In scipy this is done by giving the three parameters location, scale and shape (https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.lognorm.html). However, I do not see any equivalent in the docs for SA-lib to include that.
